workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/RadioCardList/components/RadioCard.scss

```
@@ -0,0 +1,7 @@
+.radioSelected {
```

Please, change this class name and if possible make this selector more specific.
```suggestion
.radio-selected {
```